004-governance.Rmd

```diff
@@ -76,6 +76,9 @@ They:
 
 Authors are expected to follow our standard processes, such as:
 
+-   **Welcoming and inclusive**: being friendly is a core value of the tidyverse and we strive to create an inclusive atmosphere in our GitHub interactions.
```

Comment:
@tracykteal improvements here would be particularly welcome.

Reply:
**Welcoming and inclusive**: Kindness and gratitude are core values of the tidyverse and we strive to create an [inclusive atmosphere](https://github.com/tidyverse/tidyverse.org/blob/master/CODE_OF_CONDUCT.md) in our GitHub interactions.